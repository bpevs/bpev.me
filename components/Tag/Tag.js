export default function Tag(props) {
  return <span className="tag h6 border rounded ml1">{props.children}</span>;
}
