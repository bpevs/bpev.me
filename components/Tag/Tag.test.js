import * as React from "react"
import { shallow } from "enzyme"
import Tag from "./Tag"


test("Should Render Tag", () => {
  const wrapper = shallow(<Tag>Hello</Tag>)
  expect(wrapper).toMatchSnapshot()
})
