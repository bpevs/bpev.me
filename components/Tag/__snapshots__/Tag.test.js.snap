// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Render Tag 1`] = `
<span
  className="tag h6 border rounded ml1"
>
  Hello
</span>
`;
